doc/rules/language_construct/union_null.rst

```diff
@@ -0,0 +1,61 @@
+===================
+Rule ``union_null``
```

Comment:
Let's go with the name `nullable_type`, WDYT?

Reply:
I somewhat feel like people will be looking for union when trying to find this fixer, so it'd be good to keep that word in the name.

Reply:
No, common feature of `?Foo` and `Foo|null` is nullability, not union, so if we plan to provide support for enforcing both formats (well, one of them), we need to focus on `nullable`, not on `union`. That's why `nullable_type` is better here.

Reply:
I mean, sure. I don't mind either way but I feel like people won't find it easily.